dashboard/src/modules/components/ComparisonComponent/PanelContent.jsx

```diff
@@ -21,10 +21,12 @@ const PanelConent = () => {
   } = useSelector((state) => state.comparison);
   const onFilter = useCallback(
     (item) => {
-      if (searchValue === "") {
-        return true;
+      if (item) {
```

Comment:
I suppose a passive return will be treated the same as `false` (probably `undefined`), but an explicit `return false` outside the `if` might clarify behavior.

Reply:
Got it